meshnets/tune.py

```diff
@@ -30,20 +30,20 @@
                    'The fraction of the dataset used for validation.')
 
 # Dataloaders flags
-flags.DEFINE_multi_integer('batch_size', [1, 2, 4], 'The batch size.')
+flags.DEFINE_multi_integer('batch_size', [4, 8], 'The batch size.')
 flags.DEFINE_integer('num_workers_loader', 2,
                      'The number of workers for the data loaders.')
 
 # Model parameters flags
-flags.DEFINE_list('latent_size', [8, 16, 32],
+flags.DEFINE_list('latent_size', [8, 32, 64],
```

Comment:
My OCD tells me 16 should be in this list 😅

Reply:
Added it